internal/api/handler/user_login_handler.go

```diff
@@ -20,37 +21,25 @@ func Login(ctx context.Context, profileSvc service.Service) func(http.ResponseWr
 		req, err := decodeUserLoginRequest(r)
 		if err != nil {
 			middleware.ErrorResponse(w, http.StatusBadRequest, err)
-			zap.S().Error(err)
+			zap.S().Error("Unable to decode request : ", err)
 			return
 		}
 
-		serverRequest, err := http.NewRequest("GET", os.Getenv("GOOGLE_USER_INFO_URL"), nil)
+		body, err := helpers.SendRequest(ctx, os.Getenv("GOOGLE_USER_INFO_URL"), req.AccessToken)
```

Comment:
could we make dynamic `SendRequst` function.
 Send param like methodType,url, body, header

Reply:
done